ui/app/components/dashboard/client-count-card.js

```diff
@@ -9,27 +9,22 @@ import { task } from 'ember-concurrency';
 import { waitFor } from '@ember/test-waiters';
 import { tracked } from '@glimmer/tracking';
 import { service } from '@ember/service';
-import { setStartTimeQuery } from 'core/utils/client-count-utils';
-import { dateFormat } from 'core/helpers/date-format';
+import { parseAPITimestamp } from 'core/utils/date-formatters';
 
 /**
  * @module DashboardClientCountCard
  * DashboardClientCountCard component are used to display total and new client count information
  *
  * @example
- *
- * <Dashboard::ClientCountCard @isEnterprise={{@version.isEnterprise}} />
- *
- * @param {boolean} isEnterprise - used for setting the start time for the activity log query
+ * <Dashboard::ClientCountCard />
  */
 
 export default class DashboardClientCountCard extends Component {
   @service store;
 
-  clientConfig = null;
-  licenseStartTime = null;
   @tracked activityData = null;
-  @tracked updatedAt = timestamp.now().toISOString();
+  @tracked hasActivity = null;
```

Comment:
Should this be false instead of null?

Reply:
probably good to keep it consistent for type!